src/components/activities/pharmacyActivity/pharmaceutical/components/pharmaceuticalExpiringDialog/PharmaceuticalExpiringDialog.tsx

```diff
@@ -0,0 +1,191 @@
+import { Autocomplete, TextField } from "@mui/material";
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import Button from "components/accessories/button/Button";
+import React, { FunctionComponent, useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./styles.scss";
+
+const PharmaceuticalExpiringDialog: FunctionComponent<{
+  isOpen: boolean;
+  handleSecondaryButtonClick: () => void;
+  handlePrimaryButtonClick: (
+    period: string | null,
+    month: string | null
+  ) => void;
+}> = ({ isOpen, handlePrimaryButtonClick, handleSecondaryButtonClick }) => {
```

Comment:
```suggestion
const PharmaceuticalExpiringDialog: FC<PharmaceuticalExpiringDialogProps> = ({ open, onPrimaryButtonClick, onSecondaryButtonClick }) => {
```
Extract props to an interface and use consistent names for props